2015-Fall/adrianbrink/nonFunkyCalculator/nonFunkyCalculator/ViewController.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  ViewController.swift
+//  Calculator
+//
+//  Created by aevitas on 01/09/15.
+//  Copyright (c) 2015 aevitas. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+    
+    @IBOutlet weak var display: UILabel!
+    
+    var userIsTyping: Bool = false
+    var lastAction: Bool = false
```

Comment:
Same as above with explicit type. Also I think your naming is not self explanatory. From my understanding, it is that the user has tappend on an operator.